Description: Fix documentation about exim integration

Using dspam with exim requires to call exim with option -oi to avoid mails
from being truncated. This commit modify the documentation and configuration
file to add the -oi option when calling exim.

Author: Kenneth Marshall <ktm@rice.edu>
Origin: upstream, http://dspam.git.sourceforge.net/git/gitweb.cgi?p=dspam/dspam;a=commitdiff;h=e515570c74b0e48e714c5b45f11de2a5b3482459
Bug-Debian: http://bugs.debian.org/644392
Forwarded: http://sourceforge.net/mailarchive/forum.php?thread_name=20120802190055.GF15829%40aart.rice.edu&forum_name=dspam-user
Last-Update: 2012-08-13

--- dspam-3.10.1+dfsg.orig/CHANGELOG
+++ dspam-3.10.1+dfsg/CHANGELOG
@@ -1,3 +1,8 @@
+Patch backported for Debian
+--------------
+
+[20121202:2115] sbajic: Update documentation for Exim integration.
+
 Version 3.10.2
 --------------
 
--- dspam-3.10.1+dfsg.orig/doc/exim.txt
+++ dspam-3.10.1+dfsg/doc/exim.txt
@@ -1,4 +1,4 @@
-$Id: exim.txt,v 1.1 2005/03/11 21:16:03 jonz Exp $
+$Id: exim.txt,v 1.2 2012/08/02 21:09:23 sbajic Exp $
 
 EXIM 4 INTEGRATION
 
@@ -40,11 +40,17 @@ domain as part of the username:
 
   command = /usr/local/bin/dspam --deliver=innocent --user "$local_part@$domain" -- %u
 
-Finally, you will need to configure and compile DSPAM. DSPAM will most likely
-end up calling exim again for delivery, using the spam-scanned protocol to
-identify scanned messages. The most common example is:
-
-   ./configure --with-delivery-agent="/usr/sbin/exim -oMr spam-scanned"
+Finally, you will need to configure and compile DSPAM. You can configure
+DSPAM with the appropriate LDA using --with-delivery-agent= at configure
+time or by specifying TrustedDeliveryAgent in dspam.conf. DSPAM will most
+likely end up calling exim again for delivery, using the spam-scanned
+protocol to identify scanned messages. The most common example is:
+
+  ./configure --with-delivery-agent="/usr/local/sbin/exim -oMr spam-scanned -oi"
+
+Note: DSPAM expects the LDA to NOT provide the "line with a single dot (.)"
+processing to indicate the end of data that a MTA must provide to meet the
+SMTP RFC, hence the "-oi" option to exim above.
 
 RUNNING WITHOUT PRIVILEGED EXIM USERS
 
--- dspam-3.10.1+dfsg.orig/src/dspam.conf.in
+++ dspam-3.10.1+dfsg/src/dspam.conf.in
@@ -43,7 +43,7 @@ StorageDriver @libdir@/dspam/@storage_dr
 # Other popular configurations:
 #TrustedDeliveryAgent "/usr/cyrus/bin/deliver"	# Cyrus
 #TrustedDeliveryAgent "/bin/maildrop"		# Maildrop
-#TrustedDeliveryAgent "/usr/local/sbin/exim -oMr spam-scanned" # Exim
+#TrustedDeliveryAgent "/usr/local/sbin/exim -oMr spam-scanned -oi" # Exim
 #
 TrustedDeliveryAgent "@delivery_agent@"
 
